guides/common/modules/proc_creating-hosts-with-pxeless-provisioning.adoc

```diff
@@ -11,42 +11,54 @@ For more information, see xref:Implementing_PXE_less_Discovery_{context}[].
 
 .Boot ISO Types
 
-There are four types of boot ISOs:
-
-Full host image::
-A boot ISO that contains the kernel and initial RAM disk image for the specific host.
-This image is useful if the host fails to chainload correctly.
-The provisioning template still downloads from {ProjectServer}.
-
-Subnet image::
-A boot ISO that is similar to the generic image but is configured with the address of a {SmartProxyServer}.
-This image is generic to all hosts with a provisioning NIC on the same subnet.
-The image is based on iPXE boot firmware, only limited hardware is supported.
+There are the following types of boot ISOs:
 
 ifndef::satellite[]
 Host image::
 A boot ISO for the specific host.
 This image contains only the boot files that are necessary to access the installation media on {ProjectServer}.
 The user defines the subnet data in {Project} and the image is created with static networking.
 The image is based on iPXE boot firmware, only limited hardware is supported.
+endif::[]
 
+Full host image::
+A boot ISO that contains the kernel and initial RAM disk image for the specific host.
+This image is useful if the host fails to chainload correctly.
+The provisioning template still downloads from {ProjectServer}.
+
+ifndef::satellite[]
 Generic image::
 A boot ISO that is not associated with a specific host.
 The ISO sends the host's MAC address to {ProjectServer}, which matches it against the host entry.
-The image does not store IP address details, and requires access to a DHCP server on the network to bootstrap.
+The image does not store IP address details and requires access to a DHCP server on the network to bootstrap.
 This image is also available from the `/bootdisk/disks/generic` URL on your {ProjectServer}, for example, `\https://{foreman-example-com}/bootdisk/disks/generic`.
 endif::[]
 
+Subnet image::
+A boot ISO that is not associated with a specific host.
+The ISO sends the host's MAC address to {SmartProxyServer}, which matches it against the host entry.
+The image does not store IP address details and requires access to a DHCP server on the network to bootstrap.
+This image is generic to all hosts with a provisioning NIC on the same subnet.
+The image is based on iPXE boot firmware, only limited hardware is supported.
```

Comment:
Since the Generic image is not available in the Satellite build, I tried to provide more details instead of saying that it's "similar to the Generic image".
I used this table https://github.com/theforeman/foreman_bootdisk#available-images
But it should be checked by a SME.